client/src/main/java/client/cs4224c/util/PStatement.java

```diff
@@ -25,7 +25,16 @@
     GET_BALANCE_PAYMENT("SELECT c_balance, c_ytd_payment, c_payment_cnt FROM customer WHERE c_w_id = ? AND c_d_id = ? AND c_id = ?"),
     UPDATE_BALANCE_PAYMENT("UPDATE customer SET c_balance = ?, c_ytd_payment = ?, c_payment_cnt = ? WHERE c_w_id = ? AND c_d_id = ? AND c_id = ? IF c_payment_cnt = ?"),
     GET_CUSTOMER_FULL("SELECT c_first, c_middle, c_last, c_address, c_phone, c_since, c_credit, c_credit_lim, c_discount, c_balance FROM customer WHERE c_w_id = ? AND c_d_id = ? AND c_id = ?"),
-    GET_WD_ADDRESS("SELECT w_address, d_address FROM warehouse_district WHERE d_w_id = ? AND d_id = ?");
+    GET_WD_ADDRESS("SELECT w_address, d_address FROM warehouse_district WHERE d_w_id = ? AND d_id = ?"),
+
+    //Delivery Transaction
+    GET_MIN_ORDER_ID_WITH_NULL_CARRIER_ID("SELECT dt_min_ud_o_id FROM delivery_transaction WHERE dt_w_id = ? AND dt_d_id = ?"),
+    UPDATE_MIN_ORDER_ID_WITH_NULL_CARRIER_ID("UPDATE delivery_transaction SET dt_min_ud_o_id = ? WHERE dt_w_id = ? AND dt_d_id = ?"),
```

Comment:
For all update statement, we need to give `IF xxx = xxx` to ensure that it updates based on the value we get. (To prevent some client from updating the value in between the operations)